components/BottomSheet/src/MDCBottomSheetPresentationController.m

```diff
@@ -168,7 +168,12 @@ - (void)viewWillTransitionToSize:(CGSize)size
 }
 
 - (void)updatePreferredSheetHeight {
-  CGFloat preferredContentHeight = self.presentedViewController.preferredContentSize.height;
+  CGFloat preferredContentHeight;
+  if (self.preferredSheetHeight > 0.f) {
+    preferredContentHeight = self.preferredSheetHeight;
+  } else {
+    preferredContentHeight = self.presentedViewController.preferredContentSize.height;
+  }
```

Comment:
Can you help me better understand when someone would use `preferredSheetHeight` instead of adjusting `self.presentedViewController.preferredContentSize`?